src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import React, { ChangeEvent, FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
```

Comment:
Don't forget about past checklists
```suggestion
function getUserById(id: number): User | null {
```

Reply:
I changed name of function, but I couldn't change 'userId' to 'id'. Is it a mistake?